Packs/CommonScripts/Scripts/TimeStampCompare/TimeStampCompare.py

```diff
@@ -0,0 +1,63 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa: E402 lgtm [py/polluting-import]
+from CommonServerUserPython import *  # noqa: E402 lgtm [py/polluting-import]
+
+from dateutil.parser import parse
+
+
+EQUAL = 'equal'
+BEFORE = 'before'
+AFTER = 'after'
+
+TIMESTAMP_FORMAT = '%Y-%m-%dT%H:%M:%S%Z'
+DT_STRING = "TimeStampCompare(val.TestedTime && val.TestedTime == obj.TestedTime && " \
+            "val.ComparedTime && val.ComparedTime == obj.ComparedTime)"
+
+
+def time_stamp_compare_command(args):
+    tested_time = args.get('tested_time')
+    values_to_compare = argToList(args.get('values_to_compare'))
+
+    results = []
+    parsed_tested_time = parse(tested_time)
```

Comment:
what about cases like 2/11/2020 > 1/12/2020? shouldn't we add date-format args?

Reply:
@dantavori exactly we need to cover all possible timestamp formats